common/centraldashboard/overlays/application/application.yaml

```diff
@@ -5,12 +5,12 @@ metadata:
 spec:
   selector:
     matchLabels:
-      app.kubernetes.io/name: 
-      app.kubernetes.io/instance: centraldashboard
+      app.kubernetes.io/name: centraldashboard
+      app.kubernetes.io/instance: centraldashboard-v0.5.0
```

Comment:
Why v0.5.0? Should it be 0.7.0 so we don't have to change it in the upcoming release?